content/docs/csidriver/_index.md

```diff
@@ -21,7 +21,6 @@ The CSI Drivers by Dell EMC implement an interface between [CSI](https://kuberne
 | Ubuntu        |       20.04      |       20.04         |       18.04, 20.04      |        18.04, 20.04      |          20.04     |
 | CentOS        |     7.8, 7.9     |      7.8, 7.9       |     7.8, 7.9     |      7.8, 7.9     |     7.8, 7.9     |
 | SLES          |        15SP3        |        15SP3        |       15SP3      |         15SP3     |       15SP3      |
-| Fedora Core OS|        no        |         5.x       |        no        |         no        |        no        |
```

Comment:
Verify your branch is rebased with latest from the release-2.1 branch. There is a commit and changes in this PR that are not related to the observability changes.

Reply:
Rebased with release-1.2 branch